### PR TITLE
Add migrate-routes and create-routes actions to the REPL

### PR DESCRIPTION
**Pull request by danpersa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +66,14 @@ Running it from lein:
     # Delete route
     $ ./instaskip.sh -t token-user~1-employees-route.admin delete-route -i 28
 
+    # Create route
+    $ ./instaskip.sh -t token-user~1-employees-route.admin -T pathfinder create \
+      -R 'pathfinder_InnkeeperTest: Path("/innkeeper-innkeeper-innkeeper") && Host(/^(m-at[.]release[.]zalando[.]net)$/) -> modPath(".*", "/") -> "https://www.host.com:8888";'
+
+    # Start the REPL
+    $ ./instaskip.sh -t token-user~1-employees-route.admin repl
+    # create -T pathfinder -R pathfinder_InnkeeperTest:Path("/innkeeper-innkeeper-innkeeper")&&Host(/^(m-at[.]release[.]zalando[.]net)$/)->modPath(".*","/")->"https://www.host.com:8888";
```
Comment (aryszka) [inline README.md]: do we handle multiline-commands, e.g. like using '\' before newline?

File: src/instaskip/actions/migrate.clj [hunk 2]
```diff
@@ -139,3 +139,9 @@
 
   ([routes-dir innkeeper-config]
    (migrate routes-dir (team-names-in-dir routes-dir) innkeeper-config)))
+
+
+(comment
+  (migrate "/Users/dpersa/Prog/mosaic/mosaic/routes"
```
Comment (aryszka) [inline src/instaskip/actions/migrate.clj]: local path in the opensource project

Comment (danpersa) [inline src/instaskip/actions/migrate.clj]: fixed

File: README.md (hunk 1 above)
Comment (danpersa) [inline README.md]: this version does not handle spaces and new lines in the REPL :(

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,24 +1,17 @@
...
-- Files from the new template.
-- Widget maker public API - `make-widget-sync`.
+## [0.3.2] - 2016-08-01
+### Changed
+- First stable version.
 
-[Unreleased]: https://github.com/your-name/instaskip/compare/0.1.1...HEAD
-[0.1.1]: https://github.com/your-name/instaskip/compare/0.1.0...0.1.1
+[Unreleased]: https://github.com/zalando-incubator/instaskip/compare/0.3.2...HEAD
+[0.3.2]: https://github.com/your-name/instaskip/compare/0.3.1...0.3.2
```
Comment (mo-gr) [inline CHANGELOG.md]: your-name?

Comment (danpersa) [inline CHANGELOG.md]: fixed

**Comment by NorthFury** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Pull request merged**